SUMMARY.md

```diff
@@ -34,6 +34,7 @@
 
 * [Protocol Kit](safe-core-sdk/protocol-kit/README.md)
   * [Reference](safe-core-sdk/protocol-kit/reference/README.md)
+    * [Migrating to v1](safe-core-sdk/protocol-kit/reference/migrating/v1.md)
```

Comment:
The last question, we should probably switch the order. WDYT?

Reply:
With only 2 versions people wouldn't get lost, but I think I get your point.

Will do it 👍